src/components/decks/Deck.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div
+    class="border border-gray bg-white mt-4 mb-4">
+    <div class="p-2 text-xs">
+      <div class="m-0 font-bold text-xl">
+        <router-link :to="linkTarget" class="text-black">{{ deck.title }}</router-link>
+        <span class="float-right">
+          <deck-dice :dice="deck.dice" />
+        </span>
+      </div>
+      <div class="clear-right">
+        <span class="text-sm">
+          {{ deck.user.username }}
+        </span>
+        <span class="text-sm float-right">
+          Last updated: {{ lastUpdatedDateFormatted }} ago
+        </span>
+      </div>
+      <hr />
+      <div class="mt-1 mb-1">
+        <span class="text-lg">
+          <card-link :card="deck.phoenixborn"></card-link>
+        </span>
+        <span class="text-sm float-right">
```

Comment:
If the card count isn't 30, please style the text in an error color or something (this is possible for private deck listings).

Reply:
Done!